core/src/test/scala/org/http4s/blaze/util/ExecutionSpec.scala

```diff
@@ -48,6 +48,7 @@ class ExecutionSpec extends Specification {
       i must be equalTo(10)
     }
 
+    /* This Job Can't Fail As the i+=1 is Dead Code
```

Comment:
Would you like to fix it instead of just commenting it out? I think the goal was to be able to submit a job that is going to fail anyway, so removing the counter and just ending the test with `ok` is probably fine.